pandas/core/generic.py

```diff
@@ -10501,7 +10501,7 @@ def rolling(
         cls.rolling = rolling
 
         @doc(Expanding)
-        def expanding(self, min_periods=1, center=False, axis=0):
+        def expanding(self, min_periods=1, center=None, axis=0):
```

Comment:
you can  warn right here, and if its None, don't pass it thru, just set it to False